projects/views.py

```diff
@@ -19,17 +21,27 @@
     ProjectSerializer, ProjectControlListSerializer,
 )
 
+n_completed = Cast(Count("to_project", filter=Q(to_project__status="complete")), FloatField())
+total_controls = Cast(Count("to_project"), FloatField())
+
+project_queryset = (
+        Project
+        .objects
+        .annotate(percent_complete=(n_completed / total_controls) * 100)
+        .order_by("pk")
+    )
```

Comment:
Annotating the QuerySet I think should be more efficient that implementing this calculation as a property, which may be important when there are a larger number of projects